packages/angular-cli/blueprints/module/index.js

```diff
@@ -58,7 +58,9 @@ module.exports = {
   },
 
   afterInstall: function (options) {
-    options.entity.name = path.join(this.dasherizedModuleName, this.dasherizedModuleName);
+    var componentPath = path.join(this.generatePath, this.dasherizedModuleName);
```

Comment:
@Brocco the `dasherizedModuleName` is already included in `generatePath`, so it was still there twice.

However, I still changed it to use `options.flat = false;`, without using `this.dasherizedModuleName`.

The outcome is the same though. Not sure why I need to change the test.